example/effective-dart.yaml

```diff
@@ -0,0 +1,160 @@
+# List of rules following the Effective Dart guide.
+# https://www.dartlang.org/guides/language/effective-dart
+
```

Comment:
Can we call out the incompleteness / in-progress-ness of this list?  Similar to what we're doing here:

https://github.com/dart-lang/linter/blob/97a6ec9e061c9bd0c0d85f7587f78d856ecb8ece/example/google.yaml#L7

(Possibly also in the `README`?)

Reply:
Sure, I will add the notes.